packages/eslint-config-sentry-react/index.js

```diff
@@ -18,4 +18,17 @@ module.exports = {
     // highlights literals in JSX components w/o translation tags
     'getsentry/jsx-needs-il8n': ['off'],
   },
+
+  /**
+   * Our current version of eslint unfortunately does not support extends in
+   * overrides. Once we upgrade we can uncomment this and remove local
```

Comment:
Do you know what version supports it?

Reply:
Looks like it was introduced in v6.0.0-alpha.2
https://github.com/eslint/eslint/releases/tag/v6.0.0-alpha.2
This PR: https://github.com/eslint/eslint/pull/11554